style.css

```
@@ -1,4 +1,53 @@
 * {
     margin: 0;
```

seria bom resetar o padding tb.
```suggestion
    margin: 0;
    padding: 0;
```